src/graphql/query/lendByCategory/categoryServiceLoanChannels.graphql

```diff
@@ -16,6 +16,7 @@ query loanChannelData(
 			loanChannelId
 			name
 			description
+			url
```

Comment:
Where does this get used?

Reply:
https://github.com/kiva/ui/blob/main/src/pages/Lend/LendByCategoryPage.vue#L250

The category object is based on this query which is then passed into the category row component. If the url is present the Row's header becomes a link and an additional "view more ##category name## loans" card is added at the end of the carousel.